Modules/Project/Resources/views/index.blade.php

```diff
@@ -2,36 +2,48 @@
 @section('content')
 
 <div class="container" id="vueContainer">
-    @include('project::menu_header')
-    <br>
-    <div class="d-flex justify-content-between mb-2 align-items-center">
-       <div class="d-md-flex">
-            <h4 class="">{{ config('project.status')[request()->input('status', 'active')] }} Projects
-                ({{ $projects ? $projects->count() : ''}})</h4>
-            <form class="mx-md-3" action="{{ route('project.index', ['status' => 'active'])  }}">
+    <div class='d-none d-md-flex justify-content-between'>
+        @include('project::menu_header')
+        @can('projects.create')
+            <span class='mt-4'>
+                <a href= "{{ route('project.create') }}" class="btn btn-info text-white">{{ __('Add new project') }}</a>
+            </span>
+        @endcan
+    </div>
+    <div class="mb-2">
+        <form class="d-md-flex justify-content-between ml-md-3" action="{{ route('project.index', ['status' => 'active'])  }}">
+            <div class='d-flex justify-content-between align-items-md-center mb-2 mb-xl-0'>
+                <h4 class="">{{ config('project.status')[request()->input('status', 'active')] }} Projects
+                    ({{ $projects ? $projects->count() : ''}})</h4>
                 <input type="hidden" name="status" value="{{ request()->input('status', 'active') }}">
-                <select class="fz-14 fz-lg-16 p-1 bg-info text-white rounded border-0" name="projects"
+                <select class="fz-14 fz-lg-16 p-1 bg-info ml-3 my-auto text-white rounded border-0" name="projects"
                     onchange="this.form.submit()">
                     <option value="my-projects" {{ request()->get('projects') == 'all-projects' ? '' : 'selected' }}>
                         {{ __('My Projects') }} </option>
                     <option value="all-projects" {{ request()->get('projects') == 'all-projects' ? 'selected' : '' }}>
                         {{ __('All Projects') }} </option>
                 </select>
-            </form>
-        </div>
+            </div>
+            <div class="d-flex align-items-center">
+                <input type="text" name="name" class="form-control" id="name" placeholder="Project name"
+                value=@if(request()->has('name')){{request()->get('name')}}@endif>
```

Comment:
See if this works:
```suggestion
                value={{request()->get('name')}}>
```

If not, maybe try this:
```suggestion
                value={{request()->get('name', '')}}>
```